API/contactsAPI.ts

```diff
@@ -14,7 +14,7 @@ export function fetchContacts(): Promise<Contact[]> {
         });
         resolve(data);
       } else {
```

Comment:
This will display mock contacts also in the case where user denies app access to contacts. I think we should check that the platform is also web to be safe. In the future some logic/UI will be required for when user denies access.
```suggestion
      } else if (Platform === 'web') {
```